docbrown/src/core/tgraph.rs

```diff
@@ -883,6 +885,37 @@ impl TemporalGraph {
             })
             .collect()
     }
+
+    pub(crate) fn vertex_timestamps(&self, src: u64) -> Vec<i64> {
+        let src_pid = self.logical_to_physical[&src];
+        self.timestamps[src_pid].iter().map(|t| *t).collect()
```

Comment:
Do we need to iter on this?

Reply:
self.timestamps[src_pid] is a BTreeSet<i64> which is not an iterator so needs .iter